src/properties.cpp

```diff
@@ -933,71 +933,1521 @@ namespace Exiv2 {
                                                                                                                          "level of a top-down geographical hierarchy.") },
-        { "CountryCode",        N_("Country Code"),                "closed Choice of Text",     xmpText, xmpExternal, N_("(legacy) Code of the country the content is focussing on -- either the country shown in visual "
-                                                                                                                         "media or referenced in text or audio media. This element is at the top/first level of "
+        { "CountryCode",        N_("Country Code (legacy)"),       "closed Choice of Text",     xmpText, xmpExternal, N_("(legacy) Code of the country of the location shown in the image. This element is at the top/first level of "
                                                                                                                          "a top-down geographical hierarchy. The code should be taken from ISO 3166 two or three "
-                                                                                                                         "letter code. The full name of a country should go to the \"Country\" element.") },
+                                                                                                                         "letter code. The full name of a country should go in the \"Country\" element.") },
         // End of list marker
         { nullptr, nullptr, nullptr, invalidTypeId, xmpInternal, nullptr }
     };
 
     extern const XmpPropertyInfo xmpIptcExtInfo[] = {
-        { "AddlModelInfo",           N_("Additional model info"),           "Text",                       xmpText, xmpExternal, N_("Information about the ethnicity and other facts of the model(s) in a model-released image.") },
-        { "OrganisationInImageCode", N_("Code of featured Organisation"),   "bag Text",                   xmpBag,  xmpExternal, N_("Code from controlled vocabulary for identifying the organisation or company which is featured in the image.") },
+        { "AddlModelInfo",           N_("Additional model information"),    "Text",                       xmpText, xmpExternal, N_("Information about the ethnicity and other facets of the model(s) in a model-released image.") },
+        { "OrganisationInImageCode", N_("Code of featured Organisation"),   "bag Text",                   xmpBag,  xmpExternal, N_("Code from a controlled vocabulary for identifying the organisation or company which is featured in the image.") },
+        { "AboutCvTerm",             N_("CV-Term About Image"),             "bag CVTerm",                 xmpBag,  xmpExternal, N_("One or more topics, themes or entities the content is about, each one expressed by a term from a Controlled Vocabulary.") },
```

Comment:
Should this be `AboutCvTerms` (with s at the end)? 
I noticed that it appears with an `s` in this link:
https://www.iptc.org/std/photometadata/specification/iptc-pmd-schema_2019.1.1.json

But in `https://www.iptc.org/std/photometadata/specification/iptc-pmd-techreference_2021.1.json`
I found entries with `s` and without it.

I am not sure yet how we are translating those specifications to our code structures.

Reply:
For the IPTC 2021.1 specification, I used the *XMP Specs* name rather than the *JSON Specs* name. This mapped to the XML files (i.e. *.xmp*) and seemed like the correct one to use - unless you know differently. It is worth noting that the ExifTool tags also use the *XMP Specs* name.

After your comment, I looked at the *JSON Specs* values and lots of the names are different but I don't know why.  @bquinn, can you help with this?

Reply:
Thanks for the answer. I just wanted to know how we were doing it 😉